File: api-examples/rs/count/js/spatial_circle.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://localhost:18443/resource-server/pscdcl/v1/count",
  "method": "POST",
  "headers": {
    "Content-Type": "application/json",
  },
  "processData": false,
  "data": "{\n\t\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pscdcl/aqm-bosch-climo/ABC Farm House Junction_4\",\n\t\"lat\": \"18.56581555\",\n\t\"lon\":\"73.77567708\",\n\t\"radius\":\"1\"\n}"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: api-examples/rs/search/python/spatial_circle.py
import requests

url = "https://localhost/resource-server/pscdcl/v1/search"

payload = "{\n\t\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pscdcl/changebhai/crowd-sourced-images\",\n\t\"lon\": \"18.56581555\",\n\t\"lat\":\"73.77567708\",\n\t\"radius\":\"100000\"\n}"
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data = payload)

print(response.text.encode('utf8'))
